### PR TITLE
Update footer to March 2023

### PR DESCRIPTION
**Pull request by ahmadabdolsaheb**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #XXXXX

### CODE REVIEW

File: build/universal/trending/english.yaml [hunk 1]
```diff
@@ -1,60 +1,60 @@
...
+article9link: "https://www.freecodecamp.org/news/javascript-get-current-date-todays-date-in-js/"
+article10title: "HTML Font Size"
+article10link: "https://www.freecodecamp.org/news/how-to-change-text-size-in-html/"
+article11title: "Python Lowercase"
+article11link: "https://www.freecodecamp.org/news/python-lowercase-how-to-use-the-string-lower-function/"
+article12title: "Run Python Script"
+article12link: "https://www.freecodecamp.org/news/run-python-script-how-to-execute-python-shell-commands-in-terminal/"
+article13title: "indexOf in Python"
+article13link: "https://www.freecodecamp.org/news/list-indexof-for-python-how-to-get-the-index-of-an-item-in-a-list-with-index/"
+article14title: "keyerror in Python"
```
Comment (scissorsneedfoodtoo) [inline build/universal/trending/english.yaml:29]: Currently the title for this article on News uses keyerror, but it should be KeyError.

I updated the title on Ghost, and we can update the title here:

```suggestion
article14title: "KeyError in Python"
```

File: build/universal/trending/english.yaml [hunk 2]
```diff
@@ -1,60 +1,60 @@
...
+article14link: "https://www.freecodecamp.org/news/keyerror-in-python-how-to-fix-dictionary-error/"
+article15title: "K8s vs Docker Swarm"
+article15link: "https://www.freecodecamp.org/news/kubernetes-vs-docker-swarm-what-is-the-difference/"
+article16title: "Multi-line String JS"
```
Comment (scissorsneedfoodtoo) [inline build/universal/trending/english.yaml:33]: To better match the article's title on News:

```suggestion
article16title: "Multiline String JS"
```

**Review by scissorsneedfoodtoo — APPROVED** [review]
Thanks for your patience @ahmadabdolsaheb.

Everything LGTM 👍

**Comment by ahmadabdolsaheb** [comment]
Appreciate it. I assigned the related forum footer pr to you as well.

**Pull request merged**